stdlib/proposals/task/task_listener.md

```diff
@@ -113,32 +113,8 @@ public type RecurringConfiguration record {|
 public type OneTimeConfiguration record {|
     time:Civil triggerTime;
 |};
-
-# Worker pool configuration.
-#
-# + workerCount - Specifies the number of workers that are available for the concurrent execution of jobs.
-#                 It should be a positive integer. The recommendation is to set a value less than 10. Default sets to 5.
-# + waitingTime - The number of seconds as a decimal the scheduler will tolerate a trigger to pass its next-fire-time
-#                 before being considered as `ignored the trigger`
-public type WorkerPoolConfiguration record {|
-    int workerCount = 5;
-    time:Seconds waitingTime = 5;
-|};
 ```
 
-If no worker pool is provided, the listener uses a global scheduler with the following configuration:
```

Comment:
Lets say we will be using the global schedular for the task listener and you can configure it using the global schedule ballerina configuration